crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -388,6 +393,17 @@ impl RealRibosome {
         .await?
     }
 
+    #[cfg(feature = "wasmer_sys")]
+    pub fn preserialized_module(&self, path: &PathBuf) -> RibosomeResult<Arc<Module>> {
```

Comment:
This is actually pre-compiled & pre-serialized module. Renaming this fn might be helpful